### PR TITLE
build(lerna): ask for angular changelog template

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by lionelB** [comment]
un exemple ?

**Review by lionelB — APPROVED** [review]

**Comment by douglasduteil** [comment]
> un exemple ?

https://github.com/SocialGouv/code-du-travail-numerique/pull/624

File: lerna.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
       "skip-npm": true,
       "conventionalCommits": true,
       "message": "chore(release): version %v"
+    },
+    "version": {
+      "changelog-preset": "angular"
```
Comment (lionelB) [inline lerna.json:9]: > By default, the changelog preset is set to angular. In some cases you might want to change either use a another preset or a custom one.

source : https://github.com/lerna/lerna/tree/master/commands/version#--changelog-preset

Comment (douglasduteil) [reply lerna.json:9]: The generated logs proves that its not the case :(
I guess that the documentation is on lerna is not up to date :recycle:

Comment (lionelB) [reply lerna.json:9]: alrite !

**Comment by douglasduteil** [comment]
Thanks

**Pull request merged**